plugin.json

```diff
@@ -119,6 +119,50 @@
             "type": "number", 
             "help_text": "The maximum number of users that may connect their MS Teams account. Once connected, the user is added to a white-list and may reconnect at any time.",
             "default": 1000
+        },{
+            "key": "syntheticUserAuthService",
+            "display_name": "Synthetic User Auth Service",
+            "type": "dropdown",
+              "help_text": "Select the authentication service to use when creating synthetic users. This should match the service used for member user access to Mattermost. Default is 'SAML'.",
+              "default": "saml",
+              "options": [
+                  {
+                      "display_name": "SAML",
+                      "value": "saml"
+                  },
+                  {
+                      "display_name": "AD/LDAP",
+                      "value": "ldap"
+                  }
+              ]   
+        },{
+        },{
+            "key": "syntheticUserAuthData",
+            "display_name": "Synthetic User Auth Data",
+            "type": "dropdown",
+              "help_text": "Select the MS Teams user property to use as the authentication identifier. For AD/LDAP and SAML, the identifier's value should match the value provided by the Id Attribute. ",
+              "default": "ID",
+              "options": [
+                  {
+                      "display_name": "id",
+                      "value": "ID"
+                  },
+                  {
+                      "display_name": "mail",
+                      "value": "Mail"
+                  },
+                  {
+                      "display_name": "userPrincipalName",
+                      "value": "UserPrincipalName"
```

Comment:
Aren't these opposite? I guess the display name should be value and the value should be display name for all three options.